src/backend/routers/para.test.ts

```diff
@@ -199,3 +250,25 @@ test("getMyTasks", async (t) => {
   t.is(task[0].instructions, INSTRUCTIONS);
   t.is(task[0].trial_count, TRIAL_COUNT);
 });
+
+test("getMyTasks - paras do have access", async (t) => {
+  const { trpc } = await getTestServer(t, { authenticateAs: UserType.Para });
+
+  await t.notThrowsAsync(async () => {
+    await trpc.para.getMyTasks.query();
+  });
+});
+
+test("getMyTasks - regular users don't have access", async (t) => {
+  const { trpc } = await getTestServer(t, {});
```

Comment:
See my comment on making authenticateAs default to UserType.User if not specified, and have authenticateAs just have type UserType